supreme/src/commonMain/kotlin/at/asitplus/signum/supreme/sign/Signer.kt

```diff
@@ -53,6 +56,27 @@ open class SigningKeyConfiguration internal constructor(): DSL.Data() {
          * This is treated as advisory, and may be ignored by some platforms. */
         var publicExponent: BigInteger = F4
     }
+
+    sealed class PrivateKeyConfiguration<T: CryptoPublicKey> : DSL.Data()
```

Comment:
where is this segment copied from, and why is it necessary to duplicate wherever it is copied from